supervisor-revert-to-stable.txt

```diff
@@ -0,0 +1 @@
+version=$(curl -s https://version.home-assistant.io/stable.json | jq -r '.supervisor // "latest"' || echo "latest") source /etc/os-release && docker pull "ghcr.io/home-assistant/${SUPERVISOR_ARCH}-hassio-supervisor:${version}" && docker tag "ghcr.io/home-assistant/${SUPERVISOR_ARCH}-hassio-supervisor:${version}" "ghcr.io/home-assistant/${SUPERVISOR_ARCH}-hassio-supervisor:latest" && docker stop hassio_supervisor
```

Comment:
It here a specific reason we make is hard to read?

Reply:
Good point, no real reason other than what I used for testing :sweat_smile: